tools/interactive/interactivetool_radiant.xml

```diff
@@ -0,0 +1,75 @@
+<tool id="interactive_tool_radiant" tool_type="interactive" name="radiant" version="0.1">
+    <description>Data analytics using Radiant R Shiny app</description>
+    <requirements>
+        <container type="docker">ylebras/radiant-docker</container>
+    </requirements>
+    <entry_points>
+        <entry_point name="radiant visualisation" requires_domain="True">
+            <port>3838</port>
+            <url>/sample-apps/STAT/</url>
+        </entry_point>
+    </entry_points>
+    <environment_variables>
+
+        <environment_variable name="HISTORY_ID" strip="True">${__app__.security.encode_id($outfile.history_id)}</environment_variable> <!-- FIXME: Warning: The use of __app__ is deprecated and will break backward compatibility in the near future -->
+        <environment_variable name="REMOTE_HOST">${__app__.config.galaxy_infrastructure_url}</environment_variable> <!-- FIXME: Warning: The use of __app__ is deprecated and will break backward compatibility in the near future -->
+        <environment_variable name="GALAXY_WEB_PORT">8080</environment_variable>
+        <environment_variable name="GALAXY_URL">${__app__.config.galaxy_infrastructure_url}</environment_variable> <!-- FIXME: Warning: The use of __app__ is deprecated and will break backward compatibility in the near future -->
+        <environment_variable name="API_KEY" strip="True">
+            #if $__user__:
+                #for $api_key in $__user__.api_keys:
+                    ${api_key.key}
+                    #break
+                #end for
+            #end if
+        </environment_variable> <!-- FIXME: We should have a better way to get user's API key -->
+    </environment_variables>
+    <command><![CDATA[
+
+        /usr/bin/shiny-server.sh
+
+    ]]>
+    </command>
+    <inputs>
+        <param name="infile" type="data" format="tabular,csv" label="tabular or csv file"/>
+    </inputs>
+    <outputs>
+        <!--data name="outfile" format="txt" /-->
```

Comment:
please include a output

Reply:
this is necessary ? Because for now, there is no output for galaxy.. We have to update it to use gie-helper and add outputs...

Reply:
Yeah, necessary for the moment. Please for both .... this does not mean you need to return something ... Galaxy just needs an output for the moment. So simply remove the comments.